src/runtime/tests/reference-test.ts

```diff
@@ -638,3 +639,96 @@ describe('references', () => {
     assert.sameMembers(ids, ['id:1', 'id:2']);
   });
 });
+
+async function setup(storageKeyPrefix: string | ((arcId: ArcId) => StorageKey)) {
```

Comment:
I would prefer we don't use generic setup methods like this in testing (I know there's lots of examples that do, I'm trying to get those removed too :)..)

Instead, put the setup in the top of the test. It keeps the tests more separable and clearer to read. This will also keep this test consistent with the rest of the tests in this file.

Reply:
(It's also worth pointing out that in this particular file the setup is likely to be different in each test)